content/en/docs/concepts/policy/limit-range.md

```diff
@@ -55,16 +55,18 @@ The name of a LimitRange object must be a valid
 
 A `LimitRange` does **not** check the consistency of the default values it applies. This means that a default value for the _limit_ that is set by `LimitRange` may be less than the _request_ value specified for the container in the spec that a client submits to the API server. If that happens, the final Pod will not be schedulable.
 
+
 For example, you define a `LimitRange` with this manifest:
 
-{{% code_sample file="concepts/policy/limit-range/problematic-limit-range.yaml" %}}
+{{< warning >}}
+The following examples operate within the default namespace, as the namespace parameter is undefined. This implies that any references or operations within these examples will interact with elements within the default namespace unless otherwise specified.{{< /warning >}}
```

Comment:
```suggestion
The following examples operate within the default namespace of your cluster, as the namespace
parameter is undefined and the `LimitRange` scope is limited to the namespace level.
This implies that any references or operations within these examples will interact with elements
within the default namespace of your cluster. You can override the operating namespace
by configuring namespace in the `metadata.namespace` field.
{{< /warning >}}
```
Do you agree with these?

Reply:
OK, but we'd write LimitRange without backticks.